src/__tests__/role-helpers.js

```diff
@@ -160,4 +165,18 @@ test('getImplicitAriaRoles returns expected roles for various dom nodes', () =>
   expect(getImplicitAriaRoles(input)).toEqual(['textbox'])
 })
 
+test.each([
+  ['<div hidden />', true],
+  ['<div style="display: none;"/>', true],
+  ['<div style="visibility: hidden;"/>', true],
+  ['<div aria-hidden="true" />', true],
+])('shouldExcludeFromA11yTree for %s returns %p', (html, expected) => {
```

Comment:
So far with these test cases, the function could simply be defined as: `() => true` and these tests would still pass. Could we have just one test case that returns `false`?

Reply:
Done.

Although it could still be defined as `() => (window.i = (window.i || 0) + 1) > 3` :wink: